docs/swagger.json

```diff
@@ -92,6 +92,30 @@
                 ]
             }
         },
+        "/dashboard": {
```

Comment:
we don't modify this file it should be generated from api comments. Please remove :)